content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Improving User and Developer Experience of Alertmanager UI
+---
+
+## Improving User and Developer Experience of Alertmanager UI
```

Comment:
Given no feedback from Max, I'd say we merge this for now so the schedule is accurate.